types/fakedoorcmd/message_test.go

```diff
@@ -0,0 +1,29 @@
+package commands
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const expectedMsg = `The support for List hasn't been implemented yet.
+
+If you would like us to implement it, please drop by at:
+https://github.com/hazelcast/hazelcast-commandline-client/issues/48 and add a thumbs up 👍.
+We're happy to implement it quickly based on demand!
```

Comment:
I think we decided to change this text. Could you make it shorter, maybe something like:
```
The support for List hasn't been implemented yet.
Vote for it at: https://github.com/hazelcast/hazelcast-commandline-client/issues/48
```

Reply:
You are right, I think I missed it before https://github.com/hazelcast/hazelcast-commandline-client/pull/114/commits/33e5d0a48d545a710b929043f671ab0808485bcf